docs/guide/deployment.md

```diff
@@ -108,7 +108,7 @@ If you are using the PWA plugin, your app must be served over HTTPS so that [Ser
 
    deploy:
      provider: pages
-     skip_cleanup: true
+     cleanup: true
```

Comment:
Thanks for finding this! But if I understand correctly, I think we should just remove this line, as `cleanup: true` is not the intended behavior.